src/storage/compaction_test/src/server.rs

```diff
@@ -43,8 +46,15 @@ struct Metrics {
     pub object_store_metrics: Arc<ObjectStoreMetrics>,
 }
 
-/// This tool will be started after we generate enough L0 SSTs to Hummock.
-/// Fetches and runs compaction tasks.
+/// Steps to use the compaction test tool
+///
+/// 1. Start the cluster with full-compaction-test config: `./risedev d full-compaction-test`
+/// 2. Ingest enough L0 SSTs, for example we can use the tpch-bench tool
+/// 3. Disable hummock manager commit new epochs: `./risedev ctl hummock disable-commit-epoch`,
+/// and it will print the current max committed epoch in Meta. Note that we need to wait for the
+/// frontend to unpin that epoch before going to next step.
```

Comment:
We may query status of epoch pin by rise_ctl_get_pinned_snapshots_summary.

Reply:
Or how can we know "the frontend has unpinned that epoch before going to next step"?

Reply:
I think currently we can use a poll-based method via `rise_ctl_get_pinned_snapshots_summary ` as you suggested to check that.

Reply:
I plan to submit the following PR to fix these flaws. This PR focues on performance improvement.